mk/xe-linux-distribution

```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 
-# Copyright (c) 2022, Cloud Software Group Holdings, Inc.
+# Copyright (c) {{TIME}}, {{COPYRIGHT}}
```

Comment:
TIME seems not an accurate variable name, what about `COPYRIGHTTIME` and `COPYRIGHTAUTHOR`

Reply:
Errorless